### ISSUE TITLE
Another "a valid android sdk path is required in editor settings", no valid Android SDK Path, same as #59 and #68

### ISSUE THREAD

**Issue by oneseedfruit** [abarichello/godot-ci#82]
Using image `barichello/godot-ci:3.4.4`, basically the same issue as #59 and #68. To get it to see the Android SDK path in `/usr/lib/android-sdk`, I had to run in the steps:

```
sed 's|export/android/android_sdk_path = ".*"|export/android/android_sdk_path = "'/usr/lib/android-sdk'"|g' -i /root/.config/godot/editor_settings-3.tres
mkdir -v -p ~/.config/godot
cp -v /root/.config/godot/editor_settings-3.tres ~/.config/godot/
```

Before doing this, it wouldn't detect the Android SDK path at all, but after these lines are added it worked perfectly.

**Comment by DasOhmoff** [comment]
I have this very same issue

**Comment by DasOhmoff** [comment]
I just realized #50 says that it is not possible to export mono android build. What a bummer.

**Comment by blkc** [comment]
For godot 4 is it `editor_settings-4.tres`? I am having some difficulty on the CI to have godot recognize the sdk path.

**Comment by Calinou** [comment]
> For godot 4 is it editor_settings-4.tres? 

Yes, the number in the file path corresponds to the Godot major version number.
